graylog2-web-interface/src/components/alerts/AlertTimeline.jsx

```diff
@@ -98,7 +99,7 @@ const AlertTimeline = createReactClass({
       return <Spinner />;
     }
 
-    const alert = this.props.alert;
+    const { alert } = this.props;
```

Comment:
condition and type could be merged in with this destruct and then do the conditionals below